packages/renderer/src/lib/kube/KubeApplyYAMLButton.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import Button from '../ui/Button.svelte';
+import type { KubernetesObject } from '@kubernetes/client-node';
+import SolidKubeIcon from '../images/SolidKubeIcon.svelte';
+
+let inProgress = false;
+
+async function kubeApply(): Promise<void> {
+  let contextName = await window.kubernetesGetCurrentContextName();
+  if (!contextName) {
+    return;
+  }
+
+  const result = await window.openFileDialog('Select a .yaml file to apply', {
+    name: 'YAML files',
+    extensions: ['yaml', 'yml'],
+  });
+  if (result.canceled || result.filePaths.length !== 1) {
+    return;
+  }
+
+  let file = result.filePaths[0];
+  if (!file) {
+    return;
+  }
+
+  inProgress = true;
+  try {
+    const namespace = await window.kubernetesGetCurrentNamespace();
+    let objects: KubernetesObject[] = await window.kubernetesApplyResourcesFromFile(contextName, file, namespace);
+    if (objects.length === 0) {
+      await window.showMessageBox({
+        title: 'Kubernetes',
+        type: 'warning',
+        message: `No resource(s) were applied`,
+        buttons: ['OK'],
+      });
+    } else {
+      await window.showMessageBox({
+        title: 'Kubernetes',
+        type: 'info',
+        message: `Successfully applied ${objects.length} resource(s)`,
+        buttons: ['OK'],
+      });
+    }
+  } catch (error) {
+    await window.showMessageBox({
+      title: 'Kubernetes',
+      type: 'error',
+      message: 'Could not apply Kubernetes YAML: ' + error,
```

Comment:
Better to use coercion here
```suggestion
      message: 'Could not apply Kubernetes YAML: ' + String(error),
```